config/25.7/make.conf

```diff
@@ -61,7 +61,7 @@ net_freeradius3_SET=		LDAP MITKRB_PORT MYSQL SQLITE3
 net_freeradius3_UNSET=		HEIMDAL
 net_frr10_SET=			MULTIPATH SNMP
 net_haproxy30_SET=		LUA # PROMEX
-net_miniupnpd_SET=		CHECK_PORTINUSE
+net_miniupnpd_SET=		CHECK_PORTINUSE UPNP_DISABLEPPPCONN UPNP_IGDV2 UPNP_VENDORCFG
```

Comment:
Only `UPNP_IGDV2` in the ports now so we'll park this to avoid errors during build option checks.

One thing of note is that this is usually sorted alphabetically, but we may need to work on the option names anyway. One step at a time :)